sympy/calculus/singularities.py

```diff
@@ -187,3 +187,93 @@ def is_monotonic(f, interval=S.Reals):
     dec = is_decreasing(f, interval)
 
     return fuzzy_or([inc, dec])
+
+
+def is_convergent(infinite_series, sym):
+    """
+    Convergence tests are used for checking the convergence of an
+    infinite series. There are various tests employed to check the
+    convergence. Like divergence test, root test, integral test,
+    alternating series test.
+
+    Parameters
+    ==========
+
+    infinite_series : Expr
+            An expression whose convergence is to be checked
```

Comment:
The examples below seem to indicate that ``infinite_series`` will not be a series but a term of a series. For the convergence of an infinite series it is necessary but not sufficient that the term sequence converge to zero. (That can be used as a first test to show that a series cannot be convergent.)

There is more to an infinite series than the general expression of a term and the index symbol, for example the limits that need not always be 1 and ``oo``. These could be added to the interface, but perhaps it would be better to have the whole series expression as the argument. Implemented that way ``is_convergent`` could become a method of the ``Sum`` class. This would make it easier to implement ``is_convergent`` for other classes as well.